splink/accuracy.py

```diff
@@ -191,8 +324,40 @@ def truth_space_table_from_labels_table(
 
 
 def truth_space_table_from_labels_column(
-    linker, label_colname, threshold_actual=0.5, match_weight_round_to_nearest=None
+    linker: "Linker",
+    label_colname,
+    threshold_actual=0.5,
+    match_weight_round_to_nearest=None,
+    positives_not_captured_by_blocking_rules_scored_as_zero=True,
 ):
+
+    # First we need to calculate the number of implicit true negatives
+    # That is, any pair of records which have a different ID in the labels
+    # column are a negative
+    link_type = linker._settings_obj._link_type
```

Comment:
These new calculations derive the total number labels, enabling us to compute how many 'implicit negative' (ghost labels) there are